frontend/src/style.css

```diff
@@ -445,3 +394,155 @@ list-item {
 .centerbutton {
   width: 80%;
 }
+
+/* Used in individual_form.tsx */
+.adminPane {
+  width: 100%;
+  padding: 15px 10px 5px 10px;
+  margin-bottom: 2em;
+  border: 1px solid lightgrey;
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  justify-content: space-between;
+  /* stylelint-disable */
+  background: repeating-linear-gradient(
+    135deg,
+    white,
+    white 25px,
+    rgba(0, 0, 0, 0.05) 25px,
+    rgba(0, 0, 0, 0.05) 50px
+  );
+  /* stylelint-enable */
+}
+
+/* Used in individual_form.tsx */
+.certNumber {
+  margin: 0.3em;
+}
+
+/* Used in individual_form.tsx */
+.control {
+  margin: 0.3em;
+  padding-right: 1em;
+}
+
+/* Used in individual_form.tsx */
+.controlWidth {
+  width: 50%;
+}
+
+/* Used in individual_form.tsx */
+.flexRow {
+  display: flex;
+  flex-direction: row;
+  align-items: end;
+  justify-content: start;
+}
+
+/* Used in individual_form.tsx */
+.flexRowOrColumn {
+  display: flex;
+  flex-direction: column;
+  overflow-x: hidden;
+  overflow-y: auto;
+}
+
+/* Used in individual_form.tsx */
+.individualForm {
+  display: flex;
+  overflow: hidden;
+  flex-direction: column;
+}
+
+/* Used in individual_form.tsx */
+.formPane {
+  border-right: none;
+  width: 100%;
+  padding-right: 5px;
+}
+
+/* Used in individual_form.tsx */
+.formPane:last-child {
+  padding-left: 5px;
+  padding-right: 0;
+  border-right: none;
+}
+
+/* Used in individual_form.tsx */
+.paneTitle {
+  position: absolute;
+  top: 0;
+  left: 10px;
+}
+
+/* Used in individual_form.tsx */
+.wideControlInd {
+  margin: 5px 0;
+  min-width: 195px;
+  width: 100%;
+  padding-right: 5px;
+}
+
+/* desktop */
+@media screen and (min-width: 660px) {
+  .main {
+    margin-left: 20px;
+    margin-right: 20px;
+  }
+
+  .menuButton {
+    font-size: 1.2em;
+  }
+
+  .trigram {
+    font-size: 2em;
+    margin-top: -7px;
+  }
+
+  .euText {
+    font-size: 1em;
+  }
+
+  .logo {
+    font-size: 4em;
+    color: #222;
+  }
+  /* stylelint-disable */
+  body {
+    background: #ddd center / cover no-repeat url("/images/background.jpg")
+      fixed;
+    padding: 0 20px;
+  }
+
+  .formPane {
+    border-right: 1px solid lightgrey;
+  }
+
+  .formPane {
+    border-right: 1px solid lightgrey;
+  }
```

Comment:
This is a duplicate to the rows above.

Reply:
Thanks, I think I might have forgotten to run the linter in my pc, that detects this duplication, weird that did not jumped on the github linter